rented-properties/src/main/java/org/egov/cpt/repository/PropertyRowMapper.java

```diff
@@ -173,7 +176,7 @@ private void addChildrenToProperty(ResultSet rs, Property property) throws SQLEx
 						.relationship(rs.getString("ng_relationship")).violations(rs.getString("ng_violations"))
 						.description(rs.getString("ng_description")).demandNoticeFrom(rs.getLong("ng_demandNoticeFrom"))
 						.demandNoticeTo(rs.getLong("ng_demandNoticeTo")).recoveryType(rs.getString("ng_recoveryType"))
-						.amount(rs.getDouble("ng_amount")).propertyImageId(rs.getString("ng_p_image_id"))
+						.amount(Double.valueOf(decimalFormat.format(rs.getDouble("ng_amount")))).propertyImageId(rs.getString("ng_p_image_id"))
```

Comment:
Why did we need to do this? We will lose precision.  I don't think we should do this. .

Reply:
bug is raised onit thats why, if its not need then plz let me know i will rollback

Reply:
What is the bug? What ever the bugs is.. it should be changed here. There may be other places it needs change like when convert to json.